### PR TITLE
Add the GeoTrust root as a trusted root certificate.

### PR DESCRIPTION
**Pull request by worktycho**

### CODE REVIEW

File: src/Protocol/MojangAPI.cpp [hunk 1]
```diff
@@ -43,6 +43,27 @@ Downloaded from http://certs.starfieldtech.com/repository/ */
...
+		"1XupGc1V3sjs0l44U+VcT4wt/lAjNvxm5suOpDkZALeVAjmRCw7+OC7RHQWa9k0+"
+		"bw8HHa8sHo9gOeL6NlMTOdReJivbPagUvTLrGAMoUgRx5aszPeE4uwc2hGKceeoW"
+		"MPRfwCvocWvk+QIDAQABo1MwUTAPBgNVHRMBAf8EBTADAQH/MB0GA1UdDgQWBBTA"
+		"ephojYn7qwVkDBF9qn1luMrMTjAfBgNVHSMEGDAWgBTAephojYn7qwVkDBF9qn1l"
+		"uMrMTjANBgkqhkiG9w0BAQUFAAOCAQEANeMpauUvXVSOKVCUn5kaFOSPeCpilKIn"
+		"Z57QzxpeR+nBsqTP3UEaBU6bS+5Kb1VSsyShNwrrZHYqLizz/Tt1kL/6cdjHPTfS"
+		"tQWVYrmm3ok9Nns4d0iXrKYgjy6myQzCsplFAMfOEVEiIuCl6rYVSAlk6l5PdPcF"
+		"PseKUgzbFbS9bZvlxrFUaKnjaZC2mqUPuLk/IH2uSrW4nOQdtqvmlKXBx4Ot2/Un"
+		"hw4EbNX/3aBd7YdStysVAq45pmp06drE57xNNB6pXE0zX5IJL4hmXXeXxx12E6nV"
+		"5fEWCRE11azbJHFwLJhWC9kXtNHjUStedejV0NxPNO3CBWaAocvmMw==""
```
Comment (bearbin) [inline src/Protocol/MojangAPI.cpp]: One too many "s here.

Comment (bearbin) [inline src/Protocol/MojangAPI.cpp]: Also, there are no \ns at the end of the certificate. Does that make any difference?

**Comment by SphinxC0re** [comment]
Has someone tested this?

**Comment by worktycho** [comment]
It's not a fix, but progress towards a fix.

**Comment by SafwatHalaby** [comment]
@worktycho , What's missing?

**Comment by worktycho** [comment]
We don't know yet, but think it has something to do with the wildcard.

**Comment by SafwatHalaby** [comment]
You mentioned that @madmaxoft is the one who introduced the first root cert, perhaps he knows?

**Comment by madmaxoft** [comment]
I've got this working, will publish a PR soon

**Pull request closed without merging**